src/Web/Controllers/EnvironmentController.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Contoso.FraudProtection.Web.Controllers
+{
+    public class EnvironmentController : Controller
+    {
+        [HttpPost]
+        public IActionResult Update(string env)
+        {
+            HttpContext.Session.SetString("envId", env);
+            return RedirectToAction("Index", "Catalog");
```

Comment:
Any reason you can't redirect them to whatever page they were already on?

Reply:
as discussed, not making this change.